tests/ergo_run_test.go

```diff
@@ -542,3 +551,223 @@ func TestRunOSX(t *testing.T) {
 		t.Fatalf("Expected \"ergo test response\" as response and got: %s\r\n", rez)
 	}
 }
+
+func TestConfigDynamicWindows(t *testing.T) {
+	if *getOS() != "windows" {
+		t.Skip("Not running windows run specific tests")
+	}
+	err := setupErgo()
+
+	if err != nil {
+		t.Fatalf("Could not perform setup for windows. Got %s", err.Error())
+	}
+
+	defer cleanSetup()
+	//start a web server that will initialy have no mapping in our proxy
+	startDynamicTestWebServer()
+
+	//run ergo
+	cmd := ergo("run")
+	defer func() {
+		if cmd == nil {
+			os.Exit(1)
+			return
+		}
+
+		if cmd.Process != nil {
+			cmd.Process.Kill()
+		}
+	}()
+
+	go func() {
+		cmd.Run()
+	}()
+
+	//wait for it to be sure it started
+	time.Sleep(2 * time.Second)
+
+	//we should not have a correct response
+	rez, err := getCommandResult("powershell", "-c", "$proxy=[System.Net.WebRequest]::GetSystemWebProxy();",
+		"$webclient=new-object system.net.webclient;", "$webclient.proxy=$proxy;",
+		"$webclient.DownloadString('http://dynamic.dev')")
+
+	//on windows we should get an error here
+	if err == nil {
+		t.Fatal("Expected error while requesting http://dynamic.dev, Got nil\r\n")
+	}
+
+	//make sure that we rewrite the config with the original values
+	fileContent, err := ioutil.ReadFile("./.ergo")
```

Comment:
We could use a copy of the file. Let say copy the file to `tmp/.ergo` and use the temporary file. It would avoid the rewriting of the file

Reply:
Yes, you are right. This could be done. Another option is to write the data in the temporary file at in a function and call the function at the beginning of each test.
Should we also do this modification in this PR?

Reply:
Do you think it may be an issue to Hacktoberfest?